exec/java-exec/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -199,15 +199,17 @@ SqlNode SqlCreateOrReplaceView() :
  * Parses a drop view statement.
  * DROP VIEW view_name;
  */
-SqlNode SqlDropView() :
+SqlNode SqlDropViewIfExists() :
 {
     SqlParserPos pos;
+    boolean viewExistenceCheck = false;
 }
 {
     <DROP> { pos = getPos(); }
     <VIEW>
+    [ <IF> <EXISTS> { viewExistenceCheck = true; } ]
     {
-        return new SqlDropView(pos, CompoundIdentifier());
+        return new SqlDropView(pos, CompoundIdentifier(), viewExistenceCheck);
```

Comment:
I did accordingly to `SqlCreateView` (without `OrReplace`) but agree with new names. Done.